qt/python/mantidqt/widgets/sliceviewer/test/test_sliceviewer_lineplots.py

```diff
@@ -72,6 +74,28 @@ def test_plot_with_no_line_present_creates_line_artist(self):
         self.axy.plot.assert_called_once_with(y, x, scaley=False)
         self.axy.set_ylabel.assert_called_once_with(self.image_axes.get_ylabel())
 
+    def test_plot_lines_are_set_to_width_of_below_default(self):
+        plotter = LinePlots(self.image_axes, self.mock_colorbar)
+        self.axx.set_xlabel.reset_mock()
+        x, y = np.arange(10.), np.arange(10.) * 2
+
+        plotter.plot_x_line(x, y)
+        self.axx.plot.assert_called_once_with(x, y, scalex=False)
+        self.assert_that_a_call_has_been_made_to_this_object_containing_linewidth_and_half(self.axx.plot)
+
+        self.axy.set_ylabel.reset_mock()
```

Comment:
This should be it's own test.

Reply:
It's changing the behavior of a set of tests that already exists, I respectfully disagree as this is the file dedicated to testing those line plots.

Reply:
I mean this test does one thing with `plot_x_line` then resets to do something with `plot_y_line`. They should be done in two separate units.

Reply:
Ah I understand now, my apologies. The the resets in that function are not actually necessary upon review, I have split it in 2 anyway because I think it is clearer that there are 2 individual actions occurring here, good spot.